core/admin/admin-notices.php

```diff
@@ -341,45 +340,6 @@ private function notice_role_manager_promote() {
 		return true;
 	}
 
-	private function notice_experiment_promotion() {
-		$notice_id = 'experiment_promotion';
-
-		if ( ! current_user_can( 'manage_options' ) || User::is_user_notice_viewed( $notice_id ) ) {
-			return false;
-		}
-
-		$experiments = Plugin::$instance->experiments;
-		$is_all_performance_features_active = (
-			$experiments->is_feature_active( 'e_optimized_css_loading' ) &&
```

Comment:
Please keep the notice and change it to: element caching & control loading experiments